orc8r/cloud/go/services/streamer/integ_test.go

```diff
@@ -117,7 +113,7 @@ func TestStreamerClient(t *testing.T) {
 
 	streamerClient := streamer_client.NewStreamerClient(mockedCloudRegistry{})
 	mockProvider := &mockStreamProvider{name: testStreamName, retVal: expected}
-	streamer_test_init.StartNewTestProvider(t, mockProvider)
+	streamer_test_init.StartNewTestProvider(t, mockProvider, mockProvider.name)
```

Comment:
Related to my above comment: this should just be `t, mockProvider, testStreamName`